packages/circuit-ui/components/NotificationBanner/NotificationBanner.tsx

```diff
@@ -181,24 +181,26 @@ export const NotificationBanner = forwardRef<
         className={clsx(classes.base, classes[variant], className)}
         {...props}
       >
-        <div className={classes.content}>
-          <Headline as="h2" className={classes.headline}>
-            {headline}
-          </Headline>
-          {body && <Body className={classes.body}>{body}</Body>}
+        <div className={classes.grid}>
```

Comment:
I needed to re-add a container `div` since container queries need a parent container as reference (in this case, `.base`)